### ISSUE TITLE
Howto examples or reference implementations?

### ISSUE THREAD

**Issue by clemsos** [likecoin/iscn-specs#7]
Hello, I am looking into the ISCN specs and it seems very interesting. Is there any existing implementation I could look into? Many thanks !

**Comment by githubbbie** [comment]
Yes, very cool!

**Comment by githubbbie** [comment]
@ckxpress

**Comment by benhylau** [comment]
Same here. I'd like to see examples of a blog post for example, what its ISCN looks like and how it is generated, and registered to the Likecoin registry.

**Comment by githubbbie** [comment]
Yes, @clemsos has a time estimate for this work. Should we speedup on implementation for an announcement?
